modules/apps/liferay-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,111 @@
+
+body {
+	  font-family:helvetica;
```

Comment:
We should provide fall back font options in case the client's browser does not have ```Helvetica``` installed.

Ex.
```
font-family: Helvetica, Arial, sans-serif;
```